TOOLS/osxbundle.py

```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 from __future__ import print_function
```

Comment:
If you're fully porting to python3, then you can remove this line. And also it might be good to switch `optparse` to `argparse`.

Reply:
As mentioned by @Akemi there is an existing PR about this exact change (#8735)